### PR TITLE
db connection string from env variables

### PR DESCRIPTION
**Pull request by ShilpaGhanashyamGore**
Database connection string is constructed from environment variables (in their absence default values are taken)

### CODE REVIEW

**Review by kordianbruck — APPROVED** [review]
:shipit:

**Comment by Henni** [comment]
this broke docker-compose....
I'll fix it, but please actually test the changes in the future before merging!

**Comment by ShilpaGhanashyamGore** [comment]
I did not expect that as I did not change files that docker-compose needs.
postgresConfig.json still exists .

File: index.js [hunk 1]
```diff
@@ -62,23 +62,54 @@ module.exports = {
...
+          dbConnection += ":" + databaseParams.port;
+        }
+        if(databaseParams.collection !== undefined && databaseParams.collection !== ""){
+          dbConnection += "/" + databaseParams.collection;
+        }
+        context.pgConnectionString = dbConnection;
+        console.log("CONNECTING TO " + dbConnection);
 
         //initalize Sequelize and create tables
-        context.sequelize = new context.Sequelize(config.database, config.username, config.password, {
```
Comment (Henni) [inline index.js:74]: @ShilpaGhanashyamGore why did you remove these parameters and instead handle this yourself in the lines above?

Comment (ShilpaGhanashyamGore) [reply index.js:74]: We are now going to read it from configuration file config.js.template . Earlier we read from postgresConfig.json from config folder. Parameters are before source file is different .
We either construct a db connection string fom environment variables or use the one in config.js.template .

Comment (Henni) [reply index.js:74]: But why do you create the connection string (`dbConnection`) yourself?
Sequelize does this automatically for us if we pass the `database`, `username` and `password` string to it.
